rad/src/operators/array.rs

```diff
@@ -58,6 +58,36 @@ pub fn map(input: &RadonArray, args: &[Value]) -> Result<RadonTypes, RadError> {
     Ok(RadonArray::from(result).into())
 }
 
+pub fn transpose(input: &RadonArray) -> Result<Vec<Vec<RadonTypes>>, RadError> {
```

Comment:
If the input is `&RadonArray`, shouldn't the output be `Result<RadonArray, RadError>`?

Reply:
And add tests please, especially for the error cases